src/test/java/seedu/address/storage/JsonSerializableTutorsPetTest.java

```diff
@@ -41,8 +60,27 @@ public void toModelType_invalidStudentFile_throwsIllegalValueException() throws
     }
 
     @Test
-    public void toModelType_duplicateStudents_throwsIllegalValueException() throws Exception {
-        JsonSerializableTutorsPet dataFromFile = JsonUtil.readJsonFile(DUPLICATE_STUDENT_FILE,
+    public void toModelType_duplicateStudentsNameAndTelegram_throwsIllegalValueException() throws Exception {
+        JsonSerializableTutorsPet dataFromFile = JsonUtil.readJsonFile(DUPLICATE_STUDENT_NAME_AND_TELEGRAM_FILE,
+                JsonSerializableTutorsPet.class).get();
+        assertThrows(IllegalValueException.class, JsonSerializableTutorsPet.MESSAGE_DUPLICATE_STUDENT,
+                dataFromFile::toModelType);
+    }
+
+    @Test
+    public void toModelType_duplicateStudentsNameAndEmail_throwsIllegalValueException() throws Exception {
+        JsonSerializableTutorsPet dataFromFile = JsonUtil.readJsonFile(DUPLICATE_STUDENT_NAME_AND_EMAIL_FILE,
+                JsonSerializableTutorsPet.class).get();
+        assertThrows(IllegalValueException.class, JsonSerializableTutorsPet.MESSAGE_DUPLICATE_STUDENT,
+                dataFromFile::toModelType);
+    }
+
+    /**
+     * Prevents Tutor's Pet from booting up if two or more {@code Student}s are found with the same {@code UUID}.
+     */
```

Comment:
Perhaps it will be a good idea to keep the formatting consistent with the subsequent tests?
```suggestion
    /**
     * Ensures that Tutor's Pet will not be able to boot up if two or more {@code Student}s are found with the same {@code UUID}.
     */